File: uri1117.js
let input = require('fs').readFileSync('stdin', 'utf8');
let lines = input.split('\n');
let i = 0
console.log(lines);
// notas validas = [0,10]
const notasValidas = []
while (notasValidas.length < 2)  {
    if(lines[i] >= 0 && lines[i] <= 10) notasValidas.push(lines[i]);
    else console.log("nota invalida");
    i+=1;
}
let media = (parseFloat(notasValidas[0]) + parseFloat(notasValidas[1]))/2;
// media =  notasValidas[0] + notasValidas[1];

console.log(`media = ${media}`);

File: uri1117.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float notaValida = 0;
    float somaNotaValida = 0;
    int i = 0;
    while(i < 2){
        scanf("%f",&notaValida);
        if (notaValida >= 0 && notaValida <= 10){
            somaNotaValida += notaValida;
            i++;
        }else{
            printf("nota invalida\n");
        }
    }
    printf("media = %.2f\n",somaNotaValida/2);
    return 0;
}
